File: circle/src/main/java/com/changanford/circle/utils/GlideImageLoader.java
package com.changanford.circle.utils;

import android.content.Context;
import android.graphics.Bitmap;
import android.widget.ImageView;


import com.bumptech.glide.Glide;
import com.changanford.circle.widget.assninegridview.AssNineGridView;
import com.changanford.common.utilext.GlideUtils;

/**
 * @author assionhonty
 * Created on 2018/9/19 10:29.
 * Email：assionhonty@126.com
 * Function:
 */
public class GlideImageLoader implements AssNineGridView.ImageLoader{

    @Override
    public void onDisplayImage(Context context, ImageView imageView, String url) {
//        Glide.with(context).load(url).into(imageView);
        GlideUtils.INSTANCE.loadBigImage(url,imageView);
    }

    @Override
    public Bitmap getCacheImage(String url) {
        return null;
    }
}

File: circle/src/main/java/com/changanford/circle/utils/GlideImageLoader.kt
package com.changanford.circle.utils

import android.content.Context
import com.changanford.circle.widget.assninegridview.AssNineGridView
import com.changanford.common.utilext.GlideUtils
import android.graphics.Bitmap
import android.widget.ImageView
import com.changanford.common.utilext.GlideUtils.loadCompress
import com.changanford.common.utilext.GlideUtils.loadCompress2

/**
 * @author assionhonty
 * Created on 2018/9/19 10:29.
 * Email：assionhonty@126.com
 * Function:
 */
class GlideImageLoader : AssNineGridView.ImageLoader {
    override fun onDisplayImage(context: Context, imageView: ImageView, url: String) {
//        Glide.with(context).load(url).into(imageView);
        imageView.loadCompress2(url)
    }

    override fun getCacheImage(url: String): Bitmap? {
        return null
    }
}